File: uwp/ArtistWayUWP/Models/ArtistDateEntry.cs
namespace ArtistWayUWP.Models
{
    public sealed class ArtistDateEntry
    {
        public bool Done { get; set; }
        public string Idea { get; set; } = "";
    }

    // Uma linha do histórico de Artist Dates (Recursos -> Histórico) —
    // igual ArtistDateEntry, mas com a semana (WeekStart) junto, já que o
    // histórico lista várias semanas de uma vez.
    public sealed class ArtistDateHistoryItem
    {
        public string WeekStart { get; set; }
        public bool Done { get; set; }
        public string Idea { get; set; } = "";
    }
}

File: android/ArtistWayAndroid/app/src/main/java/com/rodcarvalho/artistway/data/model/ArtistDateEntry.kt
package com.rodcarvalho.artistway.data.model

data class ArtistDateEntry(
    val done: Boolean = false,
    val idea: String = "",
)

// Uma linha do histórico de Artist Dates (Recursos -> Histórico) — igual
// ArtistDateEntry, mas com a semana (weekStart) junto, já que o histórico
// lista várias semanas de uma vez.
data class ArtistDateHistoryItem(
    val weekStart: String,
    val done: Boolean = false,
    val idea: String = "",
)
